src/components/Creators/Testimonials.vue

```diff
@@ -0,0 +1,312 @@
+<template>
+  <section class="bg-white dark:bg-darkPrimary overflow-hidden text-center px-8">
+    <h2 class="mx-auto w-full font-display text-5xl m-8 dark:text-white">Community Partners</h2>
```

Comment:
~~Community Partners~~ Testimonials

Reply:
Good catch, @vinzvinci. 👀
```suggestion
    <h2 class="mx-auto w-full font-display text-5xl m-8 dark:text-white">Testimonials</h2>
```

Reply:
This is what happens when you have two gitpod tabs open together 🤦